rotkehlchen/data_migrations/migrations/migrations_15.py

```diff
@@ -27,34 +27,38 @@ def data_migration_15(rotki: 'Rotkehlchen', progress_handler: 'MigrationProgress
 
     Redecodes the distinct Add liquidity events for each Hop LP token."""
     progress_handler.set_total_steps(1)
-
     # find one Add liquidity event for each Hop LP token
-    event_data = []
-    with rotki.data.db.conn.write_ctx() as write_cursor:
-        for identifier, db_tx_hash, db_location in write_cursor.execute(
```

Comment:
Ah i see. You were using the cursor in the loop and looping its result, and then reusing it inside the loop, which would result in the cursor being off and perhaps loop not looping all or nor deleting all.

Weird that no linter failed there. I recall this being linter-detectable.

Nice.